requirements.txt

```diff
@@ -1,10 +1,10 @@
 # Copyright 2017 Canonical Ltd.
 # Licensed under the LGPLv3, see LICENCE file for details.
 
-requests==2.18.4
-PyNaCl==1.1.2
-pymacaroons==0.12.0
-six==1.11.0
-protobuf==3.4.0
-pyRFC3339==1.0
-pytz==2017.2'
+requests>=2.18.4
```

Comment:
I think you want to preclude major version changes, for example:

    requests >=2.18.4, ==2.*

You could also use:

    requests ~= 2.18.4

but by my reading of https://www.python.org/dev/peps/pep-0440/#compatible-release, that would lock to requests 2.18, which is probably not necessary.